karma/test-directives.coffee

```diff
@@ -110,5 +115,60 @@ describe('directives:', () ->
       expect(item2.querySelector('.pod-item-value').textContent)
         .toContain('Corge')
     )
+
+    it('should draw the pod actions', ->
+      $rootScope.podData.actions = [{
+        case_id: 21,
+        type: 'foo',
+        name: 'Foo',
+        payload: {bar: 'baz'}
+      }, {
+        case_id: 21,
+        type: 'quux',
+        name: 'Quux',
+        payload: {corge: 'grault'}
+      }]
+
+      el = $compile('<div cp-pod></div>')($rootScope)[0]
+      $rootScope.$digest()
+
+      action1 = el.querySelectorAll('.pod-action')[0]
+      action2 = el.querySelectorAll('.pod-action')[1]
+
+      expect(action1.textContent).toContain('Foo')
+      expect(action2.textContent).toContain('Quux')
+    )
+
+    it('should call trigger() when an action button is clicked', ->
+      $rootScope.podData.actions = [{
+        type: 'foo',
+        name: 'Foo',
+        case_id: 21,
+        payload: {a: 'b'}
```

Comment:
I was initially thinking we could do something like that, but on second thought, I don't think it needs to. We already know the case id on the client side, so we can just send it when we make action requests. Maybe that means the format of the body needs to change a bit. Maybe something like:

```http
POST /case/action/<pod_id>/

{
    "case_id": 21,
    "action": {
      "type": "foo",
      "payload": {...}
    }
}
```

Thoughts?